### PR TITLE
Release notes for PAQ 10.16.0.2 and 10.16.0.3 fixes

### PR DESCRIPTION
**Pull request by kailol-sag**
The release notes for 10.16.0.2 and 10.16.0.3 can both go live when https://itrac.eur.ad.sag/browse/PAB-3949 moves to RELEASED-CONTROLLED.

### CODE REVIEW

**Review by skom-sag — APPROVED** [review]

**Comment by kailol-sag** [comment]
Hm, then let me also add the release notes for the PAQ 10.16.0.3 on this PR. Then both release notes can later go out at the same time.

File: content/release-10-16-0/streaming-analytics-10-16-0-bundle/10_16_0_2.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Apama-ctrl microservice</td>
+<td style="text-align:left">The <code>/prometheus</code> endpoint of the Apama-ctrl microservices is now
+restricted to only the bootstrap tenant and internal monitoring from Cumulocity IoT Operations.
```
Comment (BeateRixen) [inline content/release-10-16-0/streaming-analytics-10-16-0-bundle/10_16_0_2.md:29]:
```suggestion
restricted to only the bootstrap tenant and internal monitoring from Cumulocity IoT operations.
```

File: content/release-10-16-0/streaming-analytics-10-16-0-bundle/10_16_0_3.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<tr>
+<td style="text-align:left">Analytics Builder</td>
+<td style="text-align:left">Deploying a model in test mode and then undeploying it could fail and could cause failures for other model deployments
+if another model with the same output blocks was already deployed in production mode and formed a chain with another already deployed model.
+This issue has now been resolved.</td>
+<td style="text-align:left">PAB-3666</td>
+</tr>
+<tr>
+<td style="text-align:left">Apama runtime</td>
+<td style="text-align:left">Starting with 10.16, the Cumulocity IoT platform performs more strict validations on incoming fields of email requests.
```
Comment (BeateRixen) [inline content/release-10-16-0/streaming-analytics-10-16-0-bundle/10_16_0_3.md:35]:
```suggestion
<td style="text-align:left">Starting with release 10.16, the Cumulocity IoT platform performs more strict validations on incoming fields of email requests.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by kailol-sag** [comment]
Hi  @BeateRixen , the 10.16.0.3 fix has been published today on eu-latest. So can you please merge this PR, cherry-pick it into main, and publish it as soon as possible? Thanks, Karin

**Comment by BeateRixen** [comment]
> Hi @BeateRixen , the 10.16.0.3 fix has been published today on eu-latest. So can you please merge this PR, cherry-pick it into main, and publish it as soon as possible? Thanks, Karin

Sure, will do now.

**Pull request merged**